workbench-core/accounts/src/services/hostingAccountService.ts

```diff
@@ -17,7 +18,11 @@ export default class HostingAccountService {
   }
 
   public async get(accountId: string): Promise<Account> {
-    return await new HostingAccountLifecycleService().getAccount(accountId, true);
+    return await new HostingAccountLifecycleService().getAccount(accountId, false);
```

Comment:
No need to async/await here, can just return promise

Reply:
Hmm, was this change on purpose? Why the change from `true` to `false`?